docs/vars.md

```diff
@@ -81,7 +81,7 @@ following default cluster parameters:
   bits in kube_pods_subnet dictates how many kube_nodes can be in cluster. Setting this > 25 will
   raise an assertion in playbooks if the `kubelet_max_pods` var also isn't adjusted accordingly
   (assertion not applicable to calico which doesn't use this as a hard limit, see
-  [Calico IP block sizes](https://docs.projectcalico.org/reference/resources/ippool#block-sizes).
+  [Calico IP block sizes](https://docs.projectcalico.org/reference/resources/ippool#block-sizes)).
```

Comment:
I am not sure why adding more `)`

Reply:
This symbol has no ending
<img width="855" alt="image" src="https://user-images.githubusercontent.com/47143494/215403933-54be80d1-8ab7-440b-9a21-e5d7ec965610.png">

Reply:
Yes it's the end parenthesis for the `(assertion...` part, you are right